### ISSUE TITLE
[EU©] EU© reform might impose copyright on snippets from press websites

### ISSUE THREAD

**Issue by jibe-b** [searx/searx#1043]
DISCLAIMER: I am by no mean a lawyer, but a contributor to Searx concerned by that Searx instances continue offering the best to their users. Here is a summary of what I understand. Please help making it as accurate and up to date as possible.

Dear Searx developers, maintainers and friends,

## EU© reform

The Proposal for a Directive of the European Parliament and of the Council on copyright in the Digital Single Market, and in particular _Article 11_,

to which I will refer as "EU© reform" as I only write about the copyright questions at stake,

will affect Searx and search engines.

## Read it by yourself 

2016 version :

https://ec.europa.eu/digital-single-market/en/news/proposal-directive-european-parliament-and-council-copyright-digital-single-market

Compromise by the Council Estonian presidency (August 2017):

http://www.statewatch.org/news/2017/aug/eu-council-copyright-directive-estonian-compromises-11783-17.pdf

a partial "too long, don't read" summary of the compromise: https://juliareda.eu/2017/08/council-presidency-copyright/ Please make your opinion by reading the full compromise document

## TL;DR

The EU (c) reform stipulates that copyright owners will have a right to claim their right on press content, hence on snippets that search engines display,

even though in contravention with the Bern Convention, which keep press items outside of the most restrictive copyright rules

>Berne Convention, Article 2(8)
The protection of this Convention shall not apply to news of the day or to miscellaneous facts having the character of mere items of press information.

source : https://www.law.cornell.edu/treaties/berne/2.html

As a consequence Searx instance owners will have to conform to the rules of the press publishers: pay a license, or face court and fine and possibly takedown.

Other possibility is simply that search engines which searx take as sources no longer provide the snippets, downgrading the experience of the user searching for information.

This extension of rights of copyright holders would introduce a preceeding, an other types of contents (including scientific knowledge) might follow. _Edit_: The European Parliament Committee for Research, Industry and Energy proposed in an opinion paper dating from August 2017 to consider all scientific publications as press content, hence giving a right of exclusivity of diffusion of scientific publications for which the author has signed a copyright transfer agreement.

in the text: https://hyp.is/HP4KrJ4LEeeILb_gmO686Q/www.europarl.europa.eu/sides/getDoc.do?pubRef=-%2f%2fEP%2f%2fNONSGML%2bCOMPARL%2bPE-592.363%2b03%2bDOC%2bPDF%2bV0%2f%2fEN

Note: The Bern convention is an international Treaty on copyright that was signed in 1887 https://en.wikipedia.org/wiki/Berne_Convention

## Calendar :

Vote of the European Parliament will probably occur in November of December 2017.

## And then?

Here is an exposition of the concerns that I, and some Science organizations have http://libereurope.eu/wp-content/uploads/2017/09/Open-letter-Copyright-Reform_5Sept2017.pdf

There may be a need to act upon these issues. I will restrict myself to objective exposition of concerns.
